RyftCore/Source/Utility/CurrencyUtility.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+public final class CurrencyUtility {
+
+    public static func minorUnits(from alphabeticCurrencyCode: String) -> Int {
+        switch alphabeticCurrencyCode {
+        case "BIF", "CLP":
+            return 0
+        case "AFN", "ALL", "COP", "IDR", "IRR", "KPW", "LAK", "LBP",
+            "MMK", "MGA", "PKR", "RSD", "SYP", "SOS", "SLL", "YER":
+            return 2
+        case "BHD", "IQD":
+            return 3
+        default:
+            return currencyFormatter(
```

Comment:
for some reason `maximumFractionDigits` returns the wrong value for the currency codes I've matched above, i.e. not as you'd expect from the ISO spec